.codesandbox/ci.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packages": [
+    "packages/autocomplete-core",
+    "packages/autocomplete-js",
+    "packages/autocomplete-preset-algolia"
+  ],
+  "sandboxes": ["vanilla"]
```

Comment:
Can we use [`/examples/js`](https://github.com/algolia/autocomplete.js/tree/next/examples/js) here?

Reply:
ah yeah. it should work. let me try.